core/arch/arm/tee/pta_socket.c

```diff
@@ -0,0 +1,330 @@
+/*
```

Comment:
@etienne-lms sounds reasonable.
I was thinking about the gprof pseudo-TA that I am about to merge (PR #1209) -- I've put it in `core/arch/arm/sta`. I suppose it can fit with your description ("test oriented").